Sources/MapboxCoreNavigation/RouteController.swift

```diff
@@ -235,6 +236,7 @@ open class RouteController: NSObject {
     var previousArrivalWaypoint: MapboxDirections.Waypoint?
     
     public func locationManager(_ manager: CLLocationManager, didUpdateLocations locations: [CLLocation]) {
+        guard !hasFinishedRouting else { return }
```

Comment:
It won't since we've unsetting a route beforehand, and simulation only takes place when there is an active route. Also, if there is an active free drive session, it will feed it with updates as necessary.